spec/update/job_spec.rb

```diff
@@ -88,16 +89,24 @@ def dependencies(name, version = "1.0", platform = "ruby")
       gem_exists?(db, 'foo', '1.0', 'java')
     end
 
-
     it "saves checksum for info endpoint content" do
-      payload = BundlerApi::GemHelper.new("foo1", Gem::Version.new("1.0"), 'ruby')
-      job = BundlerApi::Job.new(db, payload, mutex, counter)
-      job.run
+      versions = %w(1.0 1.2 1.1)
+      versions.each do |version|
+        payload = BundlerApi::GemHelper.new("foo1", Gem::Version.new(version), 'ruby')
+        job = BundlerApi::Job.new(db, payload, mutex, counter)
+        job.run
 
-      get '/info/foo1'
-      checksum = Digest::MD5.hexdigest(last_response.body)
-      gem = get_gem(:info_checksum, db, 'foo1')
-      expect(gem[:info_checksum]).to eq(checksum)
+        get '/info/foo1'
+        last_response.body
+        checksum = Digest::MD5.hexdigest(last_response.body)
+
+        database_checksum = get_gem(:info_checksum, db, 'foo1', version)[:info_checksum]
+        expect(database_checksum).to eq(checksum)
+      end
+
+      expect(db[:rubygems]
+        .join(:versions, rubygem_id: :id)
+        .where(name: "foo1").order_by(:created_at).map { |v| v[:number] }).to eq(versions)
```

Comment:
Why did you add this here? Test says that `BundlerApi::Job saves checksum for info endpoint content`, when this expect block tests whether all the versions exist in database. :confused: